plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -166,4 +169,17 @@ public IcebergConfig setProjectionPushdownEnabled(boolean projectionPushdownEnab
         this.projectionPushdownEnabled = projectionPushdownEnabled;
         return this;
     }
+
+    public Optional<String> getHiveCatalogName()
+    {
+        return hiveCatalogName;
+    }
+
+    @Config("iceberg.hive-catalog-name")
```

Comment:
I'm thinking it may make more sense for this to be in a configuration specific to `IcebergHiveMetastoreCatalogModule`. If you're using a JDBC or REST Catalog implementation, it doesn't make sense to redirect to Hive right?

Granted, we don't have those yet

Reply:
This setting is (currently) relevant only in the context where Iceberg/Hive/Delta tables coexist in the same metastore (HMS/Glue).

Therefore I think that the current naming of the property is appropriate.

We'll probably soon have a Glue metastore module ;) for Iceberg

https://github.com/trinodb/trino/pull/10845/files#diff-fe450674b359e873eeaa6b35a57c840f5dd2c68cc688e388f0e585e3d3bca0e0R26